src/shared/components/gateway-section/index.js

```diff
@@ -49,6 +49,7 @@ class GatewaySection extends Component {
     const { isActive, inView, incompatible, inProgress, isMessageVisible, renderAnimation } = this.state
     const { messages } = this.props.intl
     const contentClasses = classNames(styles.content, {
+      [styles.notRendering]: !renderAnimation,
```

Comment:
It may feel strange to you because I think the name (`renderAnimation`) is not the proper one. I should rename it to `renderSvg` instead. 
However, I think your comment is still valid even if I change the name. I'll update the code.
Thanks!